common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,668 @@
+		1 = {  name = complexion_7             range = { 0.43 0.43 }     }
+	}
+}
+
+blood_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		20 = { 0.675 0.176 0.675 0.176 } # Pale Yellow/White
+		5 = { 0.392 0.169 0.392 0.169 } # Pale Green
+		1 = { 0.667 0.318 0.667 0.318 } # Pale Blue
+		5 = { 0.502 0.012 0.502 0.012 } # Brown/Gray
+	}
+	eye_color = {
+        1 = { 0 1 0 1 } # Black
+	}
+	hair_color = {
+		1 = { 1 0 1 0 } # Black/Gray
+		5 = { 0.129 0.043 0.137 0.110 } # Platinum Blonde & White
+		1 = { 0.102 0.886 0.102 0.886 } # Dark Brown
+		1 = { 0.153 0.875 0.153 0.875 } # Lighter Brown
+	}
+	
+	#Eyes
+	special_eyes = {
+        10 = { name = magic_eyes      range = { 0 1 } }
+    }
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.149 0.255 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_average_clothed		range = { 0 0 } }
+	}
+	
+	###Other
+	hairstyles = {
+		10 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+		1 = { name = no_hairstyles 		range = { 0.0 1.0 } }
+	}
+}
+
+sand_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		1 = { 0.224 0.129 0.298 0.239 } # Pale Yellow
+	}
+	eye_color = {
+        1 = { 0.686 0.310 0.686 0.350 } # Blue
+		1 = { 0.176 0.482 0.176 0.741 } # Yellow
+		1 = { 0.110 0.396 0.110 0.714 } # Orange/Brown
+	}
+	hair_color = {
+		1 = { 0.129 0.722 0.129 0.867 } # Brown
+		1 = { 0.153 0.537 0.153 0.682 } # Yellow
+		1 = { 0 0.643 0.043 0.800 } # Orange/Red
+	}
+}
+
+ice_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		1 = { 0.741 0.494 0.741 0.529 } # Blue
+		1 = { 0.659 0.396 0.659 0.427 } # Icy Blue
+		1 = { 0.714 0.459 0.714 0.459 } # Pale Blue
+	}
+	eye_color = {
+        1 = { 0 0.502 0 0.502 } # Blue
+	}
+	hair_color = {
+		5 = { 0.737 0.094 0.737 0.094 } # White
+		5 = { 0.749 0.510 0.749 0.686 } # Dark Blue
+		5 = { 0.741 0.310 0.741 0.765 } # Gray Blue to Dark Blue
+		1 = { 0.914 0.596 0.914 0.639 } # Purple
+		1 = { 0.584 0.510 0.584 0.620 } # Sea-Green
+		1 = { 0.867 0.322 0.867 0.376 } # Light Pink
+		1 = { 0.902 0.420 0.902 0.498 } # Pink
+	}
+	
+	#Height
+	gene_height = {
+        1 = { name = normal_height  range = { 0.77 0.8 }      }
+    }
+	
+	###Jaw
+	gene_jaw_width = {
+		1 = { name = jaw_width_pos    range = { 0.5 0.65 }    }
+	}
+	gene_bs_jaw_def = {
+		1 = { name = jaw_def_pos    range = { 0.6 1 }    }
+	}
+	
+	#Eyes
+	special_eyes = {
+        10 = { name = magic_eyes      range = { 0 1 } }
+    }
+	
+	#Chin	
+	gene_chin_width = {
+        1 = { name = chin_width_neg    range = { 0.5 1 }    }
+	}
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_triangle_half		range = { 0.6 1 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		50 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+		5 = { name = northern_hairstyles 		range = { 0.376 0.376 } }
```

Comment:
What is the name of this hairstyle?

Reply:
It's `male_hair_northern_01` and `female_hair_northern_02` if you want to add the female variant. But this gene is here mostly for the male hairstyle.

Reply:
Added in 720fc0ed42b3a6bc24797de7452df2810bb6aa73.